pages/public_cloud/public_cloud_databases/mongodb_14_deploy_with_terraform/guide.en-gb.md

```diff
@@ -0,0 +1,303 @@
+- Service level (plan);
+- version.
+
+You will also need to attach it to a `service_name`, which, in this case, matches your `project_id`.
+
+An important distinction between private network and public network deployment is that at this stage, you need to tell Terraform that each node will be part of the private network by specifying the `subnet_id` and `network_id` variables as follows:
+
+```bash
+resource "ovh_cloud_project_database" "service" {
+  service_name = var.product.project_id
+  description  = "terraform-hello-world"
+  engine       = "mongodb"
+  version      = "6.0"
+  plan         = "production"
+  flavor       = "db2-2"
+
+  nodes {
+    region = var.product.region
+    subnet_id  = openstack_networking_subnet_v2.my_subnet.id
+    network_id = openstack_networking_network_v2.my_private_network.id
+  }
+  nodes {
+    region = var.product.region
+    subnet_id  = openstack_networking_subnet_v2.my_subnet.id
+    network_id = openstack_networking_network_v2.my_private_network.id
+  }
+  nodes {
+    region = var.product.region
+    subnet_id  = openstack_networking_subnet_v2.my_subnet.id
+    network_id = openstack_networking_network_v2.my_private_network.id
+  }
+}
+```
+
+The MongoDB service should have been created and made available within the private network at this point, but connecting directly to it won’t be possible.
+For security reasons, you need to complete deployment by carrying out a final step.
+
+#### Step 4 - Setting up access restriction
+
+To access the MongoDB service, you will need to create connection permissions for IP addresses.
+
+So, add the IP range `192.168.12.0/24`, which you picked and used when you declared the private network ([Declaration step](#declaration)).
+This way, all Public Cloud instances hosted on it can directly access the MongoDB service.
+
+```bash
+resource "ovh_cloud_project_database_mongodb_user" "userjd" {
+resource "ovh_cloud_project_database_ip_restriction" "iprestriction" {
+  service_name = ovh_cloud_project_database.service.service_name
+  engine       = ovh_cloud_project_database.service.engine
+  cluster_id   = ovh_cloud_project_database.service.id
+  ip           = "192.168.12.0/24"
+}
+```
+
+Of course, you can be more restrictive and provide several IP addresses separately in /32, to target only certain authorised sources.
+
+#### Step 5 - User management
+
+To access the MongoDB service, you need to be an authenticated user.
+By default, the cluster will be delivered with the user `admin@admin`, whose password you can reset via the OVHcloud Control Panel to log in directly.
+
+You can also add another user via Terraform, with full wide read/write permissions on all databases, as follows:
+
+```bash
+resource "ovh_cloud_project_database_mongodb_user" "tf_user" {
+  service_name = ovh_cloud_project_database.service.service_name
+  cluster_id   = ovh_cloud_project_database.service.id
+  name         = "tf_mongodb_user"
+  roles        = ["readWriteAnyDatabase"]
```

Comment:
It's work with 0.35.0 but after ovh provider 0.37.0, authentication will be mandatory, so readWriteAnyDatabase@admin